test/test_lang_activation_builder.py

```diff
@@ -97,25 +124,34 @@ def test_with_custom_alias_with_builtin(self):
             [
                 LanguageDefinitionComponents(
                     alias="MY_PYTHON3",
-                    url="localzmq+protobuf:///bfsdefault/default/some_path/my_release"
-                    "?lang=python#buckets/bfsdefault/default/some_path/my_release"
-                    "/exaudf/exaudfclient_py3",
+                    url=LanguageDefinitionURL(
+                        protocol="localzmq+protobuf",
+                        chroot_path=ChrootPath(
+                            bucketfs_name="bfsdefault",
+                            bucket_name="default",
+                            path_in_bucket=PurePosixPath("some_path/my_release"),
+                        ),
+                        udf_client_path=UdfClientAbsolutePath(
+                            bucketfs_name="bfsdefault",
+                            bucket_name="default",
+                            executable=PurePosixPath(
+                                "some_path/my_release/exaudf/exaudfclient_py3"
+                            ),
+                        ),
+                        parameters=[SLCParameter(key="lang", value=["python"])],
+                    ),
                 ),
                 LanguageDefinitionComponents(
                     alias="JAVA",
```

Comment:
We call lang_act_build.add_custom_alias("JAVA", "MY_JAVA") before, but get here JAVA back